examples/server/server.cpp

```diff
@@ -3763,7 +3787,40 @@ int main(int argc, char ** argv) {
         std::vector<server_task> tasks;
 
         try {
-            std::vector<llama_tokens> tokenized_prompts = tokenize_input_prompts(ctx_server.vocab, data.at("prompt"), true, true);
+            common_chat_params chat_params;
```

Comment:
This is inside `handle_completions_impl`, which is shared among chat/completion/infill, so I think it's not the best place to put this logic here.

I'm wondering if it can be moved to inside `handle_chat_completions` instead

Reply:
Reverted back to a simpler change / smuggle format enum like the other data as suggested